server.js

```diff
@@ -138,7 +66,21 @@ app.get("/reviews/:product_id/meta", (req, res) => {
 // Adds a review for the given product
 app.post("/reviews/:product_id", (req, res) => {
   // console.log("req.body using bodyparser on review being added: ", req.body);
-  res.sendStatus(201);
+  const dummyReview = {
+      product_id: '1',
+      rating: 4,
+      date: '2020-08-28',
+      summary: 'okay product',
+      body: 'this was actually a so-so product',
```

Comment:
be sure to have more than 50 chars. The front end requires it. I know I was thinking about only displaying reviews that met this criterion.